### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by Muggyfox1**
added names

### CODE REVIEW

**Comment by bkelley01** [comment]
good job!

**Comment by nurse-the-code** [comment]
Great!

**Comment by yildirimao** [comment]
Hola!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # Group-Side-Project
 A project made by a group of 6 aspiring developers.
+
+## The developers:
+Muggyfox1
```
Comment (Muggyfox1) [inline README.md:5]: this is me

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
...
+
+(your name here)
```
Comment (nurse-the-code) [inline README.md:7]: this is Malachi

**Pull request merged**